grunt/aliases.json

```diff
@@ -2,6 +2,7 @@
   "default": ["nag"],
   "deploy": [
     "shell:git-is-clean",
+    "build",
     "shell:git-push-master",
     "shell:deploy-prepare",
     "build",
```

Comment:
The answer is yes. https://github.com/ga-wdi-boston/browser-template/blob/master/grunt/shell.js#L21